pkg/build/controller/build/build_controller.go

```diff
@@ -1418,10 +1433,33 @@ func (bc *BuildController) createBuildCAConfigMapSpec(build *buildv1.Build, buil
 				makeBuildCAOwnerRef(buildPod),
 			},
 		},
+		Data: data,
 	}
 	return cm
 }
 
+// createBuildCAData generates the data to be inserted into the ConfigMap containing certificate
+// authorities used by the build pod.
+func (bc *BuildController) createBuildCAData() map[string]string {
+	data := make(map[string]string)
+	if len(bc.additionalTrustedCA) == 0 {
+		return data
+	}
+	_, err := os.Stat(bc.additionalTrustedCA)
+	if err != nil {
+		glog.Warningf("Failed to find additional trusted CA bundle %s: %v", bc.additionalTrustedCA, err)
+		return data
+	}
+	pemData, err := ioutil.ReadFile(bc.additionalTrustedCA)
```

Comment:
we should not have to read this data on every build.  If the data changes we should expect the operator to restart the controller, so we can read it once during controller startup.